.env.example

```
@@ -4,4 +4,10 @@ VECTOR_DB_ENV=
 POSTGRES_URI=
 SECRET_KEY=
 # Comma-separated list of allowed CORS origins, e.g. http://localhost:3000,https://yourdomain.com
-CORS_ALLOW_ORIGINS=
\ No newline at end of file
+CORS_ALLOW_ORIGINS=
+
+# Web search configuration
+WEB_SEARCH_ENGINE=tavily
+TAVILY_API_KEY=your_tavily_api_key_here
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The TAVILY_API_KEY key should go before the WEB_SEARCH_ENGINE key

(UnorderedKey)